storage/src/main/java/org/apache/uniffle/storage/handler/impl/ComposedClientReadHandler.java

```diff
@@ -59,7 +74,8 @@ Tier next() {
   private final Map<Tier, Supplier<ClientReadHandler>> supplierMap = new EnumMap<>(Tier.class);
   private final Map<Tier, ClientReadHandler> handlerMap = new EnumMap<>(Tier.class);
   private final Map<Tier, ClientReadHandlerMetric> metricsMap = new EnumMap<>(Tier.class);
-  private Tier currentTier = Tier.VALUES[0]; // == Tier.HOT
```

Comment:
`Tier.VALUES[0]` means "the hottest tier", which currently is `Tier.HOT`.

Reply:
Yes, I know. Do you think there is something wrong with my comments ?

Reply:
Javadoc should be what this field mean or what it's used for.

Here the `// == Tier.HOT` comment is just for readability.

Reply:
Well, this may be caused by different coding specifications. I will change this.